bin/nbb

```diff
@@ -344,6 +339,24 @@ find_centos_7_ami_ids() {
   done
 }
 
+shutdown_instances() {
+  aws ec2 describe-instances --filters "Name=instance-state-name,Values=running" | jq -c -r '.Reservations[] .Instances[] .InstanceId' > instance_list
```

Comment:
Suggestion: invert the state name if possible to effectively say: "show me all instances that are NOT in state `terminated`". Catches any state (terminating, shutting-down, stopped, running, error, etc.) that could potentially lead to dependency violations.

**New dependency in `$PATH`:** `jq`. Remember to list it in the readme under what you very cleverly named "the shopping list" for anyone not using `brew`.

Suggestion: instead of just throwing it all into one file with a hard-coded name, it may be useful to put the instance list in a file with a date-stamped suffix, i.e. `instance-list.2016-06-03-16-55-33UTC.txt`. (_This is why I suggested tweaking the .gitignore pattern above_). With this format we can let the user decide to keep output state as reported at a given date/time. Can provide a "point in time" state snapshot that might be useful to them for unknown reasons, and will almost certainly be useful _for us_ when debugging.

Suggestion: isolate this line into a separate shell command called  `status` or something similar. Since we're just getting a list of instances running, we can pull that out into its own shell command and decouple the "discovery" responsibilty from the "destroy" responsibility. That enables us to create a `make provider-status` command (or equivalent) that could be helpful in debugging as well.